_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,243 @@
+  method with parameters still to be determined. Community feedback on activation
+  proposals continues to be collected. ([transcript][activation transcript],
+  [video][activation video])
+
+- **Signet**: Kalle Alm and AJ Towns participated in a discussion on
+  [Signet][topic signet].  The design decisions of Signet were explored as well
+  as the mechanics of how testnet and regtest work. For more details see the
+  [_Default signet discussion_ news item](#default-signet-discussion).
+  ([transcript][signet transcript], [video][signet video])
+
+- **Bitcoin Core GUI meeting**: Anonymized participants including designers and
+  developers met to discuss the Bitcoin Core graphical user interface, its
+  current state, how it could be improved or revamped and the constraints that
+  any possible revamp would be subject to. For example, one change that has been
+  discussed in the past is moving from Qt Widgets to the Qt QML framework which
+  is more flexible and easily customized. ([transcript][bitcoin core gui
+  transcript]).
+
+- **Sydney meetup discussion** Ruben Somsen and Rusty Russell participated in a
+  discussion on statechains, upgrading LN channels and [lnprototest][]. Somsen
+  outlined how statechains (see [Newsletter #100][news100 statechains]) could
+  potentially be used to swap out your Lightning channel counterparty or in the
+  case of DLCs (see [Newsletter #81][news81 dlc]) swap out a portion of your
+  position without closing the channel. With various possible Lightning channel
+  upgrades on the horizon it was speculated which could use dynamic commitments
+  (see [Newsletter #108][news108 dynamic commitments]) and which would require
+  [splicing][topic splicing]. Finally, Russell explained how the Lightning
+  protocol test suite lnprototest could be used to find bugs in existing
+  implementations and assist with ensuring interoperability across
+  implementations when building out new features.  ([transcript][sydney
+  transcript]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19797][] Remove old check for 3-byte shifted IP addresses from pre-0.2.9 nodes FIXME:dongcarl
+
+- [Bitcoin Core #19731][] extends the `getpeerinfo` RPC results with two new
+  fields: `last_block` and `last_transaction`. `last_block` is the time that
+  the node most recently received a block from the peer which it had not seen
+  before and which passed preliminary validity checks. `last_transaction` is the
+  time tht the node most recently received a transaction from the peer that it
+  accepted to its mempool. Both of these metrics are used in our peer
```

Comment:
"our" sounds off here. Perhaps remove it. s/are used in our peer/are used in peer